src/checkout-buttons/checkout-button-initializer.spec.ts

```diff
@@ -18,11 +18,12 @@ describe('CheckoutButtonInitializer', () => {
     let initializer: CheckoutButtonInitializer;
     let buttonActionCreator: CheckoutButtonStrategyActionCreator;
     let store: CheckoutStore;
+    const locale = 'en';
```

Comment:
Same here, instead of assigning to a temporary variable, you could directly pass it to the function